pandas/tests/frame/test_apply.py

```diff
@@ -1056,3 +1056,26 @@ def test_non_callable_aggregates(self):
         expected = df.size
 
         assert result == expected
+
+    @pytest.mark.parametrize("df", [
+        pd.DataFrame([[1, 2], [3, 4]]),
+        pd.DataFrame([[np.nan, 2], [3, 4]]),
+        pd.DataFrame(),
+    ])
+    def test_agg_function_input(self, df, cython_table_items):
+        # test whether the functions (keys) in
+        # pd.core.base.SelectionMixin._cython_table give the same result
+        # as the related strings (values) when used in df.agg. Examples:
```

Comment:
add an example which actually tests (say for sum, nansum) axis=1, IOW contruct the resultant frame

this test doesn't actually tests that axis=1 works, just that it matches with a string (which doesn't have tests itself)